charts/secrets-store-csi-driver-provider-aws/values.yaml

```diff
@@ -46,3 +46,6 @@ securityContext:
 useFipsEndpoint: false
 
 dnsConfig: {}
+
+secrets-store-csi-driver:
+  install: true
```

Comment:
We elected to make this default to true since we are doing a major version bump of the helm chart along with the Go SDK v2 release. This way we bundle backwards incompatible changes in the same release together while improving the customer experience by reducing the number of commands that need to be run to deploy the CSI driver.

We will update the README accordingly after the release.